packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/__init__.py

```diff
@@ -23,13 +23,15 @@ def __init__(
         upload_base64_image: Optional[
             Callable[[str, str, str, str], Coroutine[None, None, str]]
         ] = lambda *args: "",
+        disable_trace_context_propagation: bool = False,
     ):
         super().__init__()
         Config.enrich_assistant = enrich_assistant
         Config.enrich_token_usage = enrich_token_usage
         Config.exception_logger = exception_logger
         Config.get_common_metrics_attributes = get_common_metrics_attributes
         Config.upload_base64_image = upload_base64_image
+        Config.disable_trace_context_propagation = disable_trace_context_propagation
```

Comment:
nit: I think it's better to have "positive flags" - so this should be "enable_trace_context_propagation" and default to `True`. WDYT?

Reply:
sure, no problem. done :)